spec/requests/case_court_reports_spec.rb

```diff
@@ -174,18 +184,7 @@
       it "uses the custom template" do
         get JSON.parse(response.body)["link"]
 
-        expect(get_docx_subfile_contents(response.body, "word/header2.xml")).to include("Voices for Children Montgomery")
```

Comment:
We should keep this test in addition to the new test

Reply:
The new template doesn't have "Voices for Children Montgomery" and when you extract the xml from the docx there's no `word/header2.xml`

Reply:
the `as a supervisor` test is moved up and commented out